velox/vector/SimpleVector.h

```diff
@@ -149,6 +149,10 @@ class SimpleVector : public BaseVector {
     return {flags.ascending ? result : result * -1};
   }
 
+  void validate(const VectorValidateOptions& options) const override {
```

Comment:
nit : why not also put this in the cpp file..

Reply:
I don't have a strong opinion.
However it is a one-liner, I usually leave them in headers, unless they need some header inclusion. That way it is less to browse for engineers, especially useful when IDE is not great (our case ;) ).